src/test/firecracker/virtio-block-parse/main.rs

```diff
@@ -42,15 +46,23 @@ impl GuestMemoryMmap {
             if let Some(prev_addr) = TRACK_READ_OBJ {
                 assert!(prev_addr.0 != addr.0);
             }
-            if rmc::nondet() && TRACK_READ_OBJ.is_none() {
+            if rmc::any() && TRACK_READ_OBJ.is_none() {
                 TRACK_READ_OBJ = Some(addr);
             }
         }
-        rmc::nondet()
+        if rmc::any() {
```

Comment:
Is this change maintaining the old behavior?

Reply:
From what I understood... any insight?

Reply:
If we want to replace this with just:
```rust
rmc::any()
```
we would have to add an `Invariant` trait bound to `T`, is that correct? Not sure which option is better (assuming what I mentioned is correct).

Reply:
I see what you mean. This does remove some constraints from the proof. That said, the proof still succeeds, so I don't think there is any harm done. 

I'll add the Invariant and use rmc::any since that is the prefeed method.